runners/spark/pom.xml

```diff
@@ -134,6 +152,28 @@
             <forkCount>1</forkCount>
             <reuseForks>false</reuseForks>
           </configuration>
+          <executions>
+            <execution>
+              <id>runnable-on-service-tests</id>
+                 <configuration>
+                   <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
+                <parallel>none</parallel>
+                <failIfNoTests>true</failIfNoTests>
+                <dependenciesToScan>
+                  <dependency>org.apache.beam:java-sdk-all</dependency>
+                </dependenciesToScan>
+                <systemPropertyVariables>
+                  <beamTestPipelineOptions>
+                    [
+                      "--runner=org.apache.beam.runners.spark.TestSparkPipelineRunner",
+                      "--streaming=false"
+                    ]
+                  </beamTestPipelineOptions>
+                  <dataflow.spark.test.reuseSparkContext>true</dataflow.spark.test.reuseSparkContext>
```

Comment:
Since all the tests are running in the same JVM, only one context is allowed.
This should actually even make things (a little bit) faster.. since your not creating the context for each test.
Keep in mind that all tests that are running a Spark pipeline, will run a standalone Spark instance - and as you can see in the runner unit tests, it takes ~3-4 seconds..